Code/GraphMol/MolDraw2D/MolDraw2DSVG.h

```diff
@@ -86,6 +86,31 @@ class RDKIT_MOLDRAW2D_EXPORT MolDraw2DSVG : public MolDraw2D {
   void initDrawing() override;
   void initTextDrawer(bool noFreetype) override;
 
+  template <class t_obj>
+  void outputTagClasses(const t_obj *obj) const {
+    if (!d_activeClass.empty()) {
+      d_os << " " << d_activeClass;
+    }
+    if (obj->hasProp("_tagClass")) {
+      std::string value;
+      obj->getProp("_tagClass", value);
+      std::replace(value.begin(), value.end(), '\"', '_');
+      std::replace(value.begin(), value.end(), '\'', '_');
```

Comment:
Same question, why not replace with `&apos;`?

Reply:
See above